controller_names:
  - fake_panda_arm_controller
  - fake_hand_controller
fake_panda_arm_controller:
    joints:
      - panda_joint1
      - panda_joint2
      - panda_joint3
      - panda_joint4
      - panda_joint5
      - panda_joint6
      - panda_joint7
fake_hand_controller:
    joints:
      - panda_finger_joint1
      - panda_finger_joint2
initial:
   panda_arm: ready
   hand: open